src/ngraph/builder/matmul_factory.cpp

```diff
@@ -37,53 +38,43 @@ using namespace ngraph::onnx_import::matmul;
 ///
 /// \return     The node representing sub matrix.
 ///
-static std::shared_ptr<ngraph::Node> get_sub_matrix(const std::shared_ptr<ngraph::Node>& node,
-                                                    std::size_t idx)
+static shared_ptr<Node> get_sub_matrix(const shared_ptr<Node>& node, size_t idx)
```

Comment:
`shared_ptr<Node>` should be `Output<Node>` so it works with inputs that aren't output 0 of a node.

Reply:
Done